packages/integration-tests/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sentry-browser-integration-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rimraf -g suites/**/dist",
+    "install-browsers": "playwright install --with-deps",
+    "lint": "run-s lint:prettier lint:eslint",
+    "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
+    "lint:prettier": "prettier --check \"{suites,utils}/**/*.ts\"",
+    "test:ci": "playwright test ./suites --browser='all' --reporter='line'",
+    "test": "playwright test ./suites",
+    "posttest": "yarn clean"
```

Comment:
If the tests could be negatively influenced by existing/stale files under `suites/**/dist`, then we should do the `clean` step before running the tests.

(Or generate the pages/assets into a test-run-specific temp folder)